pom.xml

```diff
@@ -112,9 +107,8 @@
     <dependencies>
         <dependency>
             <groupId>org.openapitools</groupId>
-            <artifactId>openapi-generator</artifactId>
+            <artifactId>openapi-generator-cli</artifactId>
```

Comment:
If I'm understanding correctly, rather than downloading the CLI and adding it to the classpath on execution, this will declare the CLI as a dependency and package it together with the generator as an uber JAR?

Reply:
Yerp.